docs/manuals/source/Configuration/Director.rst

```diff
@@ -81,8 +81,16 @@ Only a single type (Backup, Restore, ...) can be specified for any job. If you w
 
 Note, you define only a single Job to do the Full, Differential, and Incremental backups since the different backup levels are tied together by a unique Job name. Normally, you will have only one Job per Client, but if a client has a really huge number of files (more than several million), you might want to split it into to Jobs each with a different FileSet covering only part of the total files.
 
-Multiple Storage daemons are not currently supported for Jobs, so if you do want to use multiple storage daemons, you will need to create a different Job and ensure that for each Job that the combination of Client and FileSet are unique. The Client and FileSet are what Bareos uses to restore a client, so if there are multiple Jobs with the same Client and FileSet or multiple Storage daemons that are used, the restore will not work. This problem can be resolved by defining multiple FileSet
-definitions (the names must be different, but the contents of the FileSets may be the same).
+Multiple Storage daemons are not currently supported for Jobs, if you do want to use multiple storage daemons, you will need to create a different Job and ensure that that the combination of Client and FileSet are unique.
+
+.. warning::
+
+  Bareos uses only :config:option:`dir/job/Client` and :config:option:`dir/job/Fileset` to determine which jobids belong together.
+  If job A and B have the same client and fileset defined, the resulting jobids will be intermixed:
+  When a job determines its predecessor to determine its required level and since-time, it will consider all jobs with the same client and fileset.
```

Comment:
```suggestion
  A) When a job determines its predecessor to determine its required level and since-time, it will consider all jobs with the same client and fileset.
```

Reply:
I'm not sure this will be formatted in a way I like, but I'll try it.